airbyte-integrations/connectors/source-oracle/spec.patch.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
What does this patch do? I see a couple like this with the empty properties object.

Reply:
They do nothing, these patch files are put in place by `new-connector.sh` script as a template to make it easier for people to write patches. They are no-op in their current state.